src/components/layout/LayoutMain.tsx

```diff
@@ -31,7 +32,7 @@ export const LayoutMain = ({ children }: { children: ReactNode }) => {
         </div>
       </div>
 
-      {children}
+      <Hydration>{children}</Hydration>
```

Comment:
Added this to avoid Next.js hydration errors when page reloads.

Reply:
thanks for the context!